### ISSUE TITLE
Agree on settings and security

### ISSUE THREAD

**Issue by sqldan** [CallCreditGroup/Policies#5]
2 factor authentication?
SSH Certificates?
Installing GitHub apps?
Making your organisational association public?
Do we provide a contactable email address to the outside world?
Does CallCredit have a gravatar logo?
Allowing contributors
Member privileges? e.g. all can create / delete repos.

**Comment by Arturas-K** [comment]
2 factor = must
ssh - user preference
GitHub app - user preference
Organisation membership = must, if you dont want your personal account linked, that is fine you can create new github account using work email, i would say this is preferred.
Contactable email - user preference, we have organisational one that should be public for raising issues of code of conduct violations
gravatar - no idea
Member privileges - this should be discussed under different issue #3

**Comment by sqldan** [comment]
Agreed. On Contactable email for the organisation, I don't think we have one yet. Nor would I know who this should currently go to.

On 2-Factor, is there not a way to avoid having to use 2 factor when connecting from certain locations? e.g. with a certificate?
